src/tokens/saga.ts

```diff
@@ -140,63 +133,6 @@ export async function createTokenTransferTransaction(
   )
 }
 
-export function* stableTokenTransferLegacySaga() {
-  const tag = 'stableToken/saga'
-
-  while (true) {
-    const transferAction = (yield* take(Actions.TRANSFER)) as TokenTransferAction
-    const { recipientAddress, amount, currency, comment, feeInfo, context } = transferAction
-
-    Logger.debug(
-      tag,
-      'Transferring token',
-      context.description ?? 'No description',
-      context.id,
-      currency,
-      amount,
-      feeInfo
-    )
-
-    yield* put(
-      addStandbyTransactionLegacy({
-        context,
-        type: TokenTransactionType.Sent,
-        comment,
-        status: TransactionStatus.Pending,
-        value: amount.toString(),
-        currency,
-        timestamp: Math.floor(Date.now() / 1000),
-        address: recipientAddress,
-      })
-    )
-
-    try {
-      const account: string = yield* call(getConnectedUnlockedAccount)
-
-      const currencyAddress: string = yield* call(getCurrencyAddress, currency)
-      const tx = yield* call(createTokenTransferTransaction, currencyAddress, {
```

Comment:
pull on the `createTokenTransferTransaction` string too! I was able to follow it all the way to this unused component that we can delete https://github.com/valora-inc/wallet/blob/main/src/fees/CalculateFee.tsx lots more dead code!

Reply:
TY, i didn't notice this! I have removed :D